.github/workflows/test.yml

```diff
@@ -0,0 +1,39 @@
+name: Run tests
+on: push
+
+env:
+  DATABASE_URL: postgres://root:root@localhost:5432
+
+jobs:
+  runner-job:
```

Comment:
Change name

Reply:
Better name for the file could be `test-backend.yml` since in theory the same directory could hold some other tests too.